src/Appwrite/Utopia/Database/Validator/Queries/Users/ListUsers.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Appwrite\Utopia\Database\Validator\Queries\Users;
```

Comment:
No need for the extra namespace for now.
```suggestion
namespace Appwrite\Utopia\Database\Validator\Queries;
```

Reply:
I believe this was resolved, cannot find such occurrence in the codebase.